extensions/chathistory.md

```diff
@@ -0,0 +1,158 @@
+---
+title: IRCv3 chathistory extension
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Evan Magaliff"
+    period: "2017"
+    email: "evan@muffinmedic.net"
+  -
+    name: "Darren Whitlen"
+    period: "2018-2019"
+    email: "darren@kiwiirc.com"
+---
+## Description
+This document describes the format of the `chathistory` extension. This enables clients to request messages that were previously sent if they are still available on the server.
+
+The server as mentioned in this document may refer to either an IRC server or an IRC bouncer.
+
+## Implementation
+The `chathistory` extension uses the [chathistory][batch/chathistory] batch type and introduces a client command, `chathistory`.
+
+To fully support this extension, clients MUST support the [`batch`][batch], [`server-time`][server-time] and [`message-tags`][message-tags] capabilities.
```

Comment:
Someone interpreted this language as meaning that chathistory has a hard dependency on batch, server-time, and message-tags, and therefore that chathistory could not necessarily be used to replace `znc.in/playback`.

I think it's worth clarifying that this is not the case: degraded functionality (in particular, functionality equivalent to `znc.in/playback`) is still available even without support for any of these caps.